server/build.gradle.kts

```diff
@@ -63,79 +63,91 @@ tasks.register("bumpVersion") {
     replaceVersion("gradle.properties", "(?<=^version=).+")
 }
 
+// Vals
+val dockerProjectRootDirectory: Directory = layout.projectDirectory.dir("docker")
+val dockerBuildRootDirectory: Directory = layout.buildDirectory.dir("docker").get()
+
 // Docker related tasks
+val copyDockerFolder: TaskProvider<Copy> =
+    tasks.register<Copy>("copyDockerFolder") {
+        description = "Copies the docker folder to the build root directory"
+        group = "docker"
 
-var updateDockerEnv =
-    tasks.register<Exec>("updateDockerEnv") {
-        description =
-            "Creates the .env file in the docker folder that contains environment variables for docker"
+        from(dockerProjectRootDirectory)
+        into(dockerBuildRootDirectory)
+    }
+
+// @todo(#343) - createProductionDotEnv is temporary and used by the suites,
+//  once the suites no longer rely on the .env file from the build root we
+//  should remove this task
+val createProductionDotEnv: TaskProvider<Exec> =
+    tasks.register<Exec>("createProductionDotEnv") {
+        description = "Creates the default dotenv file for the Block Node Server"
         group = "docker"
 
-        workingDir(layout.projectDirectory.dir("docker"))
+        dependsOn(copyDockerFolder)
+        workingDir(dockerBuildRootDirectory)
         commandLine("sh", "-c", "./update-env.sh ${project.version} false false")
     }
 
-tasks.register<Exec>("createDockerImage") {
-    description = "Creates the docker image of the Block Node Server based on the current version"
-    group = "docker"
+val createDockerImage: TaskProvider<Exec> =
+    tasks.register<Exec>("createDockerImage") {
+        description =
+            "Creates the production docker image of the Block Node Server based on the current version"
+        group = "docker"
 
-    dependsOn(updateDockerEnv, tasks.assemble)
-    workingDir(layout.projectDirectory.dir("docker"))
-    commandLine("./docker-build.sh", project.version, layout.projectDirectory.dir("..").asFile)
-}
+        dependsOn(copyDockerFolder, tasks.assemble)
+        workingDir(dockerBuildRootDirectory)
+        commandLine("sh", "-c", "./docker-build.sh ${project.version}")
+    }
 
 tasks.register<Exec>("startDockerContainer") {
-    description = "Starts the docker container of the Block Node Server of the current version"
+    description =
+        "Starts the docker productiongit a container of the Block Node Server for the current version"
```

Comment:
```suggestion
        "Starts the docker production container of the Block Node Server for the current version"
```

Reply:
✅ @rbarkerSL thanks, I have missed that typo. This is pushed.